### DISCUSSION TITLE
articles/sapiens

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#147] [Comments]
https://hutusi.com/articles/sapiens 

 看过《人类简史》的人都说这是一本奇书，也的确是一本奇书，书中一些观点可以说是让我刷新了三观。以前我们读到的历史书，从孔子著《春秋》起，作者总要站在一个所谓历史旁观者的角度来评判，对与错正义与否等。或者是我们在政治课中所学习的唯物主义历史观，历史是人民大众推动的，不以人的意志为转移，等等。这些观点，在《人类简史》中...

**Comment by petite-dd** [comment]
人类的发展，我也写过一篇《南北半球被某种神秘力量分隔》，看到文中人能改造人的时候确实比我想的还要远一些。我觉得下个阶段人的四肢会退化，用机械外骨骼等等手段代替，增加活动力。变成看起来头大身体四肢小，脱离科技就非常柔弱的物种
